go.sum

```diff
@@ -139,6 +143,18 @@ github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
 github.com/kr/text v0.2.0 h1:5Nx0Ya0ZqY2ygV366QzturHI13Jq95ApcVaJBhpS+AY=
 github.com/kr/text v0.2.0/go.mod h1:eLer722TekiGuMkidMxC/pM04lWEeraHUUmBw8l2grE=
+github.com/lestrrat-go/blackmagic v1.0.2 h1:Cg2gVSc9h7sz9NOByczrbUvLopQmXrfFx//N+AkAr5k=
+github.com/lestrrat-go/blackmagic v1.0.2/go.mod h1:UrEqBzIR2U6CnzVyUtfM6oZNMt/7O7Vohk2J0OGSAtU=
```

Comment:
these deps seem possibly unmaintained what is the justification for using these vs a more standard packages?

Reply:
- Library: https://github.com/lestrrat-go/jwx is maintained; I can check with regards to dependencies 

I can easily replace it with https://github.com/golang-jwt/jw

Reply:
List of company/project using "lestrrat-go/jwx": https://github.com/lestrrat-go/jwx/wiki/Users

Reply:
https://github.com/lestrrat-go/jwx does seem maintained but blackmagic does not.  We are just trying to ensure we use the best options available. I will leave that to you.

Reply:
I verified the deps are stable even if they aren't updated frequently.